src/software/ai/hl/stp/play/defense_play.cpp

```diff
@@ -59,9 +68,11 @@ void DefensePlay::getNextTactics(TacticCoroutine::push_type &yield)
                                                CreaseDefenderTactic::LeftOrRight::RIGHT),
     };
 
-    // TODO: replace with reasonable fallback tactics
+    // fallback tactics slightly more reasonable than before
```

Comment:
I'm confused about what this comment is saying or why it's here?

Reply:
me too. Remove it since it's not useful

Reply:
Comment is still here?

Reply:
removed it for real this time